audio/softsynth/mt32.cpp

```diff
@@ -169,12 +169,13 @@ int MidiDriver_MT32::open() {
 	debug(4, _s("Initializing MT-32 Emulator"));
 
 	Common::File controlFile;
-	if (!controlFile.open("CM32L_CONTROL.ROM") && !controlFile.open("MT32_CONTROL.ROM"))
-		error("Error opening MT32_CONTROL.ROM / CM32L_CONTROL.ROM. Check that your Extra Path in Paths settings is set to the correct directory");
-
 	Common::File pcmFile;
-	if (!pcmFile.open("CM32L_PCM.ROM") && !pcmFile.open("MT32_PCM.ROM"))
-		error("Error opening MT32_PCM.ROM / CM32L_PCM.ROM. Check that your Extra Path in Paths settings is set to the correct directory");
+	if (!controlFile.open("CM32L_CONTROL.ROM") || !pcmFile.open("CM32L_PCM.ROM")) {
+		debug("Error opening CM32L_CONTROL.ROM / CM32L_PCM.ROM. Falling back to MT32");
```

Comment:
That's not an error. It should be worded in a different way

```suggestion
		debug("Unable to open CM32L_CONTROL.ROM / CM32L_PCM.ROM. Falling back to MT32");
```

Reply:
As we may have 0 or 1 file out of 2, it could be interesting to warn the user about that.
Having 1 file is definitely not expected (warning) while having 0 means that the user doesn't have the ROMs for the CM32L.